src/main/java/com/smoothstack/utopiaairlines/services/AirportAdminService.java

```diff
@@ -0,0 +1,97 @@
+/**
+ * 
+ */
+package com.smoothstack.utopiaairlines.services;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import com.smoothstack.utopiaairlines.dao.AirportDao;
+import com.smoothstack.utopiaairlines.entities.Airport;
+
+@Service
+public class AirportAdminService {
+
+	@Autowired
+	private AirportDao airportDAO;
+
+	/**
+	 * Create a new airport
+	 * 
+	 * @author Josten Asercion
+	 * @param airport the new airport to be created
+	 */
+	public Airport create(Airport airport) {
+		return airportDAO.save(airport);
+	}
+
+	/**
+	 * Get all Airports in DB
+	 * 
+	 * @author Josten Asercion
+	 * @return List of all airports
+	 */
+	public List<Airport> all() {
+		return airportDAO.findAll();
+	}
+
+	/**
+	 * Return a airport specified by an id
+	 * 
+	 * @author Josten Asercion
+	 * @param id the id
+	 * @return Airport specified by the id, null if DNE
+	 */
+
+	public List<Airport> searchByIataId(String IataId) {
+		return airportDAO.findByIataIdLike(IataId);
```

Comment:
Why not use `getById(...)`? IATAs are unique as they're the primary key. No need to search for a list that will always be of size 1.